test/index.js

```diff
@@ -14,12 +14,24 @@ test('parse selector', (t) => {
   t.end()
 })
 
-test('parse selector class with attributes', (t) => {
+test('parse selector class with attributes class as string', (t) => {
   const obj = parse(['p.italic', { class: 'bold' }])
   t.equal(obj.attrs.class, 'italic bold')
   t.end()
 })
 
+test('parse selector class with attributes class as array', (t) => {
+  const obj = parse(['p.italic', { class: ['bold'] }])
+  t.deepEqual(obj.attrs.class, ['italic', 'bold'])
```

Comment:
I don't understand how `obj.attrs.class` would be anything else than a string or an object. 

based on the definitions, no array support is included in preact

  https://github.com/developit/preact/blob/523c397a8c99560fea77ef558c04beb27c99eb7c/src/preact.d.ts#L281-L282